listenbrainz_spark/stats/user/tests/test_entity.py

```diff
@@ -86,3 +86,34 @@ def test_get_entity_all_time(self, mock_create_messages, mock_get_listens, mock_
         mock_df.createOrReplaceTempView.assert_called_with('user_test_all_time')
         mock_create_messages.assert_called_with(data='user_test_all_time_data', entity='test', stats_range='all_time',
                                                 from_ts=from_date.timestamp(), to_ts=to_date.timestamp())
+
+    def test_create_messages_recordings(self):
+        """ Test to check if the number of recordings are clipped to top 1000 """
+        recordings = []
+        for i in range(0, 2000):
```

Comment:
This pattern of generating test data instead of saving huge json files in git is great! We should consider doing this on the frontend as well in the future.